wcarhart/Konphig
# Refactored architecture

Big changes for portability updates.

Resolves #2, resolves #15, resolves #17, resolves #19, resolves #20.

Things that still need to be implemented:
 * Test functions on MacOS (see #11)
 * Update documentation (`functions.md` and `README.md`) (see #16)
 * Add dependency check and/or installation to `install.sh` (see #18)
